release.go

```diff
@@ -172,3 +173,14 @@ func aggregateReleaseVersion(bundles []Bundle) (string, error) {
 
 	return version, nil
 }
+
+func GetNewestRelease(releases []Release) (Release, error) {
+	if len(releases) == 0 {
+		return Release{}, microerror.Maskf(executionFailedError, "releases must not be empty")
+	}
+
+	s := SortReleasesByVersion(releases)
+	sort.Sort(s)
+
+	return s[len(s)-1], nil
+}
```

Comment:
In our API we want to default to the newest release in case no release version is given. Right now we only select the first item of the list. See https://github.com/giantswarm/cluster-service/blob/c5b1747b1e086bbc33aebd888da31811a22c7ceb/service/release/searcher/searcher.go#L76. This function is to improve the situation.